@shared/api/helpers/extensionMessaging.ts

```diff
@@ -45,7 +40,7 @@ export const sendMessageToBackground = async (msg: {}): Promise<Response> => {
     // treat this as an external call because we're making the call from the browser, not the popup
     res = await sendMessageToContentScript(msg);
   } else {
-    res = await browser.runtime.sendMessage(msg);
+    res = await browser?.runtime.sendMessage(msg);
```

Comment:
Good call - at one point I was having issues getting the codebase to build without this (since it's building in a node environment), but this no longer seems to be an issue.

You're right, there shouldn't be a time when the app is running and browser does not exist